src/Apps/MyCollection/myCollectionRoutes.tsx

```diff
@@ -41,11 +52,6 @@ export const myCollectionRoutes: AppRouteConfig[] = [
     onClientSideRender: () => {
       MyCollectionArtwork.preload()
     },
-    prepareVariables: ({ artworkID }) => {
-      return {
-        artworkID,
-      }
-    },
```

Comment:
Why did you remove this? 👀

Reply:
It's automatically returning artworkID. This is only needed if the variables are manipulated. I thought removing it will make the file a bit smaller and easier to read.